tests/e2e/specs/in-game/reconnecting.spec.js

```diff
@@ -699,3 +699,219 @@ describe('Reauthenticating in game', () => {
     cy.get('#deck').should('be.visible');
   });
 });
+
+describe('Reconnecting after game is over', () => {
+  beforeEach(() => {
+    cy.setupGameAsP0();
+  });
+  
+  it('Dialogs persist after refreshing when game is over by conceded and opponent request rematch', () => {
+    cy.concedeOpponent();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
```

Comment:
How about throwing another `cy.reload()` here and asserting the dialog shows up before anyone rematches? Just another moment in time that someone could disconnect